### DISCUSSION TITLE
Sensore rilevatore fumo e monossido carbonio

### DISCUSSION THREAD

**Discussion by Foofighterclub** [indomus/forum#2862] [GENERALE - Discussioni generiche]
Buongiorno.
Sono possessore di una stufa a legna, e da quando la utilizzo, sento molto odore di fumo in casa.
La legna nella stufa arde bene, cerco di aprire il più possibile le finestre per il ricambio d'aria ed il camino viene pulito ogni anno. Al piano di sotto, collegato allo stesso camino, funziona un'altra stufa a legna, di diverso tipo, che non provoca lo stesso odore di fumo. 
Pensavo di acquista ed applicare, anche per controllo di sicurezza, un sensore fumo e monossido, in modo tale da verificare se la qualità dell'aria sia accettabile o meno, per capire se l'odore di fumo sia soggettivo oppure se ci sia un problema di emissioni della stufa.

Ci sono dei dispositivi, magari con connessione alla rete/app, validi da consigliarmi, tipo Google Nest Protect?
E' preferibile un sensore di qualità dell'aria, tipo Netatmo?

Grazie.

**Comment by indomus** [comment]
https://indomus.it/componenti/x-sense-xs01-wt-sensore-di-fumo-wi-fi-con-supporto-tuya/
https://indomus.it/componenti/amazon-smart-air-quality-monitor-multi-sensore-di-qualita-dellaria/

**Comment by 50494554524F** [comment]
(oltre al discorso sensori) ovviamente quando c'è qualcosa che brucia devi avere SEMPRE le finestre aperte, non aprirle il più possibile... se sono aperte basta sapere che il monossido va verso l'alto e ti puoi regolare

deto questo ci sono anche gli xiaomi honeywell; al momento ho integrato solo il fumo ed facile da integrare, il monossido devo ancora studiarlo
